docs/UserGuide.adoc

```diff
@@ -34,6 +34,8 @@ Help is also shown if you enter an incorrect command e.g. `abcd`
 Adds a person to the address book. +
 Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 
+* Alias: `a`
```

Comment:
You could explain that the command alias is a replacement, rather than a new parameter to Add command.